src/Symfony/Installer/NewCommand.php

```diff
@@ -609,4 +609,15 @@ private function getExecutedCommand()
 
         return sprintf('%s new %s %s', $executedCommand, $this->projectName, $version);
     }
+
+    /**
+     * Checks whether the given directory is empty or not.
+     *
+     * @param  string  $dir the path of the directory to check
+     * @return boolean      True if the directory is empty. False otherwise.
+     */
+    private function isEmptyDirectory($dir)
+    {
+        return 0 === count(glob($dir.'/*', GLOB_NOSORT));
```

Comment:
Hmm, I checked and you're totally right. Probably would better use this code:
```php
return 2 === count(scandir($dir.'/'));
```
For the info, 2 is the number matching an empty dir where the only accessible elements are `.` and `..` , so count two :)